api/v1alpha1/condition_consts.go

```diff
@@ -37,3 +37,19 @@ const (
 	// DeletedReason (Severity=Info) documents a condition not in Status=True because the underlying object was deleted.
 	DeletedReason = "Deleted"
 )
+
+// Conditions related to the VirtualMachineImages
+const(
+	// VirtualMachineImageOsTypeSupportedCondition denotes that the OS type in the virtual machine image object is
+	// supported by VMService or not. e.g. Image with windows OS type is not supported by VM Service.
+	VirtualMachineImageOsTypeSupportedCondition ConditionType = "VirtualMachineImageOsTypeSupported"
+	// VirtualMachineImageSupportedCondition denotes that the image compatibility with VMService. VMService expects
+	// VirtualMachineImage to be prepared by VI admin by adding "vmware-system.vmsvc.compatibility" key or a photon image.
```

Comment:
This condition name conflicts with the purpose of `ImageSupportedState` below (which would be a combination or multiple conditions). 

How about this condition be renamed `VirtualMachineImageVmwarePreparedImageCondition` or `VirtualMachineImage

Also remove the note about the actual key and VI Admins, we should just state it's a Image that's specially prepared for VM Service. The decision is still a bit in flux, but initially vmware and related vendors will create the images according to our instructions and publish with the key.